### PR TITLE
DOK viewing all scorecards

### PR DESCRIPTION
**Pull request by DavidOKeefe**
@ChaeOkay :eyes:

### CODE REVIEW

**Comment by DavidOKeefe** [comment]
@ChaeOkay :eyes:

**Comment by DavidOKeefe** [comment]
screen shot 2016-02-17 at 5 40 53 pm

screen shot 2016-02-17 at 5 41 10 pm

**Comment by ChaeOkay** [comment]
Maybe split up the nav bar links so that it doesn't look like they are on the same line.

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -1,2 +1,3 @@
 - if user_signed_in?
   = link_to(I18n.t('header.logout'), destroy_user_session_path)
+  = link_to(I18n.t('header.scorecards'), scorecards_path)
```
Comment (ChaeOkay) [inline app/views/layouts/_header.html.slim]: maybe 

```
- if user_signed_in?
  ul
    li = link_to(I18n.t('header.logout'), destroy_user_session_path)
    li = link_to(I18n.t('header.scorecards'), scorecards_path)
```

**Comment by DavidOKeefe** [comment]
@ChaeOkay :eyes: 

screen shot 2016-02-17 at 5 49 51 pm

screen shot 2016-02-17 at 5 50 07 pm

**Comment by ChaeOkay** [comment]
nice

**Pull request merged**